isites_migration/management/commands/export_slide_tool.py

```diff
@@ -67,29 +81,55 @@ def handle(self, *args, **options):
             for file_node in FileNode.objects.filter(file_repository=file_repository):
                 if file_node.file_type == 'file':
                     if file_node.storage_node:
-                        storage_node_location = file_node.storage_node.physical_location
+                        storage_node_loc = file_node.storage_node.physical_location
                     elif file_repository.storage_node:
-                        storage_node_location = file_repository.storage_node.physical_location
+                        storage_node_loc = file_repository.storage_node.physical_location
                     else:
                         logger.error("Failed to find storage node for file node %d", file_node.file_node_id)
                         continue
 
-                    file_name = os.path.basename(file_node.physical_location)
-                    s3_key = "%s/%d/%s" % (keyword, topic.topic_id, file_name)
+                    # Determine if we're dealing with an fs-docs storage node or an fs-cow
+                    if 'fs-docs' in storage_node_loc:
+                        # For pre-COW files, we need to construct the physical file location
+                        # ourselves
+                        physical_file_loc = os.path.join(
```

Comment:
Thanks for clarifying that this logic came from the predecessor of this repo (canavas_course_admin_tool)